config/src/test/java/org/springframework/security/config/http/OAuth2ResourceServerBeanDefinitionParserTests.java

```diff
@@ -486,6 +487,20 @@ public void requestWhenBearerTokenResolverAllowsRequestBodyAndRequestContainsTwo
 		// @formatter:on
 	}
 
+	@Test
+	public void requestWhenBearerTokenResolverAllowsRequestBodyAndRequestContainsTwoTokensInRequestBodyThenInvalidRequest()
+			throws Exception {
+		this.spring.configLocations(xml("MockJwtDecoder"), xml("AllowBearerTokenInBody")).autowire();
+		// @formatter:off
+		MockHttpServletRequestBuilder request = post("/authenticated")
```

Comment:
With the below suggestion, you may need to add a `Content-Type` header to this test.